pkg/sql/schemachanger/screl/attr.go

```diff
@@ -402,7 +402,7 @@ var Schema = rel.MustSchema("screl", append(
 	),
```

Comment:
While you're at this, could you also update `el.AttrType(Element, t((*protoutil.Message)(nil)).Elem()),`  at line#111 in this file to narrow `Element` attribute down to `scpb.isElementProto_ElementOneOf` interface? I think this is a good to have but not necessary. Your call.